java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -58,6 +61,16 @@ public void custom_command_ping() {
         assertEquals("PONG", data.getSingleValue());
     }
 
+    @Test
+    @SneakyThrows
+    public void custom_command_del_returns_a_number() {
+        clusterClient.set("DELME", INITIAL_VALUE).get();
```

Comment:
minor: `String key = "DELME";` and use `key` throughout

Reply:
Or use test name as the key to avoid collisions with other tests.